Create Test_Yandex.py

davydov-den opened the pull request.

davydov-den committed:
Create Test_Yandex.py

davydov-den committed:
Update Test_Yandex.py

skyline-gleb commented on 0204/Test_Yandex.py:
```diff
@@ -0,0 +1,47 @@
+        else:
+            check = 0
+        try:
+            self.assertEqual(1, check, "Неверное значение")
```
"input_text in title_value" вовзращает булевое значение которое можно сразу проверить используя подходящий ассерт.

davydov-den replied:
Исправил.

skyline-gleb commented on 0204/Test_Yandex.py:
```diff
@@ -0,0 +1,47 @@
+        try:
+            self.assertEqual("Яндекс: задан пустой поисковый запрос", title_value, "Неверное значение")
+        finally:
+            driver.close()
```
надо понимать, что блок try/finally временный, когда заиспользуешь setUp/tearDown этим блокам в тестах не место

davydov-den replied:
Запомню на будущее.

skyline-gleb commented:
в первом тесте заиспользуй еще какое-нибудь получение атрибута и его проверку, чтоб хоть немного расширить кол-во разных используемых методов

davydov-den committed:
Update Test_Yandex.py

davydov-den commented:
Добавил.

skyline-gleb commented on 0204/Test_Yandex.py:
```diff
@@ -0,0 +1,51 @@
+        element.submit()
+        
+        try:
+            self.assertEqual(1, input_text in title_value, "Неверное значение")
```
не поправил, так и остался неподходящий ассерт. В презентации была таблица возможных.
Иначе поиск в интернете если презентацию смотреть слишком просто https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue

davydov-den replied:
Заюзал другой ассерт.

skyline-gleb replied:
огонь! :+1:

skyline-gleb commented:
добавил все теже действия что и были, ничего нового. я прделагал, например, заиспользовать получение аттрибутов, а это именно получение атрибутов http://selenium-python.readthedocs.org/en/latest/api.html#selenium.webdriver.remote.webelement.WebElement.get_attribute

davydov-den committed:
Другой ассерт и получение атрибута

Добавил другой ассерт для проверки вложенности строк
Добавил тест с получение атрибута
